core/vm/actor/builtin/types/miner/v2/monies.cpp

```diff
@@ -0,0 +1,204 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "monies.hpp"
+
+#include <vm/actor/actor_method.hpp>
+#include <vm/actor/builtin/v2/miner/miner_actor.hpp>
+#include <vm/actor/builtin/v2/multisig/multisig_actor.hpp>
```

Comment:
```suggestion
#include "vm/actor/actor_method.hpp"
#include "vm/actor/builtin/v2/miner/miner_actor.hpp"
#include "vm/actor/builtin/v2/multisig/multisig_actor.hpp"
```

Use `#include "..."` for project filess, and `#include <...>` for standart and external libraries